src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function copyTabCol() {
+  const table = document.querySelector('table');
+  const rows = table.querySelectorAll('tr');
+
+  rows.forEach((row) => {
+    const cellToCopy = row.querySelector(':nth-child(2)');
+
+    const copiedCell = document.createElement(cellToCopy.tagName);
+
+    copiedCell.textContent = cellToCopy.textContent;
+
+    const cellToPaste = row.lastElementChild;
```

Comment:
Ensure that `cellToPaste` is not `null`. If a row does not have a last child, `lastElementChild` will return `null`, which will cause an error when calling `before`.